recipes/librdkafka/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.8)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+
+if(WITH_ZLIB)
+  list(APPEND test_compile_definitions WITH_ZLIB)
```

Comment:
Why do you need these definitions? I don't see any #ifdef in your test.

Reply:
I thought there should be some tests for different options. But now I think its not necessary.